### ISSUE TITLE
Как вытащить лидов?

### ISSUE THREAD

**Issue by vladstart1** [Bolid1/npm-api-client-amocrm#2]
Подскажите пожалуйста как вытащить лидов за определенную дату?

**Comment by Bolid1** [comment]
К сожалению amoCRM поддерживает фильтрацию только по дате изменения.
Подробнее об этом можно почитать в [документации](https://www.amocrm.ru/developers/content/api/leads), раздел "Список сделок"

**Comment by vladstart1** [comment]
Да, я видел эту информацияю, но я не понял как такой же фильтр задать в nodeJS.
Вот для примера как я вытягиваю список лидов:
```
amoClient.auth(subdomain, login, key).then(function (res) {
  console.log('auth res: ', res);
  if (res.auth === true) {
        amoClient.listLeads().then(function (leads) {
          console.log('lead:', leads);
        });
  }
```
Я вот не совсем разобрался где тут использовать фильтрацию по дате изменения и как ее применить. Можете пожалуйста помочь.

**Comment by Bolid1** [comment]
К сожалению на данный момент в библиотеке нет поддержки этого функционала.
Добавлю себе в список работ.
